playwright/tenor/README.md

```
@@ -0,0 +1,117 @@
+### Bosatte, myndige personer (`bosattMyndigPerson` / `bosatteMyndigePersoner`)
+| pid | navn |
+|---|---|
+| 03906197811 | Straffet Kost |
+| 23854897845 | Konservativ Fattigmannskost |
+| 13894599892 | Søt Kompetanse |
+| 50907400120 | Vassen Ert |
+| 17889574100 | Ansvarsfull Regle |
+| 09893049719 | Anstendig Purre |
+
+### Virksomhet med daglig leder (`dagligLederMedOrg`)
+| dagligLeder pid | orgnr | orgnavn |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)